src/ng/httpBackend.js

```diff
@@ -109,7 +109,12 @@ function createHttpBackend($browser, createXhr, $browserDefer, callbacks, rawDoc
         }
       }
 
-      xhr.send(post || null);
+      if (post === false) {
```

Comment:
Thanks for your guidance, @gkalpak. I was able to fix my npm installation using https://github.com/brock/node-reinstall. I updated the code and new tests to reflect what I believe we should expect. Unit tests passed and E2E may be failing for extraneous reasons as you suggested. I'll update the PR per your instructions.